### PR TITLE
chore(styles): update css pipeline to emit fallback colors for IE11

### LINKED ISSUE #29088: Stats: IE Bar Graph Missing Views Column
#### Steps to reproduce
1. Navigate to the Stats page for a blog in Internet Explorer 11 (I used Browserstack)

#### What I expected
The Views columns should display along with the Visitors column

#### What happened instead
You will see the Views (light blue) columns missing, while the Visitors (dark blue) columns remain.

#### Browser / OS version
Internet Explorer 11 

#### Screenshot / Video
ie no column

#### Context / Source
#user-report Reported by user in ticket 1612580-z

-->

### PR DESCRIPTION
**Pull request by blowery**
So far, no luck.

fixes #29088

### CODE REVIEW

**Comment by blowery** [comment]
Looks like the root (heh) of the problem is that when building the individual stylesheets for chunks, we do not have the custom properties available, as they're not imported. We don't want to import the custom property definitions from _color-schemes into every chunk, as it would lead to massive duplication.

We can provide those definitions using the `importFrom` [property](https://github.com/postcss/postcss-custom-properties#importfrom) on the postcss custom properties plugin, but we need to hand that either real CSS (not SASS) or JS or JSON. I think the easiest path forward would be to build a color-schemes.css from _color-schemes.scss as a prereq to the webpack build and use that.

**Review by flootr — COMMENTED** [review]
This fix works for me 👍 it adds the necessary fallbacks. Also tested IE11 through Browserstack and the stats charts display just fine now 🍏

File: webpack.config.js [hunk 1]
```diff
@@ -248,13 +248,13 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 					test: /\.(sc|sa|c)ss$/,
 					use: [
 						MiniCssExtractPluginWithRTL.loader,
-						'css-loader',
 						{
-							loader: 'postcss-loader',
+							loader: 'css-loader',
 							options: {
-								plugins: [ require( 'autoprefixer' ) ],
+								importLoaders: 2,
```
Comment (jsnajdr) [inline webpack.config.js]: Is the `importLoaders` option needed for anything? I tried to build CSS both with and without it and the resulting CSS assets are exactly the same. I'm very confused about what exactly all these CSS-ish loaders do. `css-loader` docs say it resolves `@import` statements, but doesn't the `sass-loader` do that for us?

**Review by jsnajdr — APPROVED** [review]
This patch works great. I verified that all CSS assets, no matter which of our several methods were used to build it, all have the color fallbacks.

File: webpack.config.js (hunk 1 above)
Comment (jsnajdr) [reply webpack.config.js]: I see that [create-react-app does the same thing](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L471).

**Comment by sirreal** [comment]
This broke the Jetpack build. It's likely that other SDK builds are broken as well.

The CI Job is unfortunately unaware that the SDK fails because it does not report a failure exit status to the shell (#30019)

**Pull request merged**